ARROW-14652: [R] Dataset vignette download script likely to fail with default options (#11656)
apache/arrow: r/vignettes/dataset.Rmd

```

If your arrow build doesn't have S3 support, you can download the files

> with the additional code shown below.  Note, that as these are large files, 
> to successfully download them, you may first need to increase your download 
> timeout from the default of 60 seconds, e.g.
> `options(timeout = 300)`.

Reviewer: ```suggestion
with the additional code shown below.  Since these are large files, 
you may need to increase R's download timeout from the default of 60 seconds, e.g.
`options(timeout = 300)`.
```

> `options(timeout = max(300, getOption("timeout")))`.

Reviewer: I get that this is a more correct solution, but if we're saying the default is 60, why not just:

```suggestion
`options(timeout = 300)`.
```

Author: Yep, that makes it simpler, thanks.
